src/Resources/Resources/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 ## Upcoming Release
 * Used utf8 encoding for reading stdout & stderr when invoking Bicep. [#23246]
 * Fixed regression in `Publish-AzBicepModule` [Azure/bicep/12461](https://github.com/Azure/bicep/issues/12461)
+* Added `-WithSource` parameter to `Publish-AzBicepModule` for publishing source with a module (currently experimental)
```

Comment:
@isra-fel, we may consider add a preview attribute for experimental parameter in our framework in future. Current preview attribute CmdletPreview is for cmdlet only.

Reply:
Hi @StephenWeatherford , how unstable is the -WithSource feature? Are you comfortable if we need it to follow the policy of breaking change? (breaking changes allowed only in major releases, major release being twice a year)

Reply:
Got it. Let's make it this way.